prboom2/src/p_setup.c

```diff
@@ -2849,12 +2849,43 @@ void P_SetupLevel(int episode, int map, int playermask, skill_t skill)
   R_SmoothPlaying_Reset(NULL);
 }
 
+//
+// P_InitThingProperties
+//
+// Initialize Thing extra properties (keeping vanilla props in info.c)
+//
+static void P_InitThingProperties (void)
```

Comment:
This loop needs to be moved to `D_BuildBEXTables()`.

Reply:
Thanks